lib/rules/bypass.json

```diff
@@ -1,7 +1,6 @@
 {
 	"id": "bypass",
 	"selector": "html",
-	"pageLevel": true,
```

Comment:
Based on our discussion (pairing), this should not be removed?

See - https://github.com/dequelabs/axe-core/blob/4b99965b40ccb0d6ecfda620c1bbd697a504aa2e/doc/API.md#L214